packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/core",
-  "version": "1.5.1",
+  "version": "1.5.2-alpha.0",
```

Comment:
Not a huge deal, but we don't need to update some of these packages, only the ones that rely on the api-sdk.

To achieve that, one can use:

```shell
npm run publish:alpha:ignore -- --force-publish @metriport/api-sdk
```

Reply:
Gotcha ill use that going forward with subsequent prs since i already updated them now